plugins/woocommerce/tests/legacy/unit-tests/log/log-handler-email.php

```diff
@@ -262,4 +289,13 @@ public function test_multiple_send_log() {
 			$this->normalize_eol( $mailer->get_sent( 1 )->body )
 		);
 	}
+
+	/**
+	 * Filters the "from" address in emails to use the expected test email.
+	 *
+	 * @return string
+	 */
+	public function filter_test_email() {
```

Comment:
This is required because when using `wp-env` the email is `admin@localhost` which isn't considered a valid email address. Technically I think it _is_ valid, but, whatever!